index.bs

```diff
@@ -917,7 +913,7 @@ and "timestamp" as arguments.
         1.  Let |permission_state| be the result of invoking
             [=request sensor access=] with |sensor_instance| as argument.
         1.  If |permission_state| is "granted",
-            1.  Invoke [=register a sensor object=] with |sensor_instance| as argument.
+            1.  Invoke [=activate a sensor object=] with |sensor_instance| as argument.
```

Comment:
[=activate a sensor object=] might be asynchronous, maybe it is better to run it as a task?